<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>saveFile</class>
 <widget class="QWidget" name="saveFile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>文件导入及存储</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,2,0">
   <property name="spacing">
    <number>20</number>
   </property>
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <property name="leftMargin">
    <number>15</number>
   </property>
   <property name="topMargin">
    <number>15</number>
   </property>
   <property name="rightMargin">
    <number>15</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item>
    <widget class="QGroupBox" name="inputFile">
     <property name="title">
      <string>输入文件</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <property name="topMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>9</number>
      </property>
      <item row="0" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="12,0">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="0,2">
          <property name="spacing">
           <number>5</number>
          </property>
          <item>
           <widget class="QLabel" name="fileLable">
            <property name="text">
             <string>文件路径</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="fileEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="0">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetFixedSize</enum>
          </property>
          <item>
           <widget class="QPushButton" name="fileOptionBnt">
            <property name="text">
             <string>...</string>
            </property>
            <property name="iconSize">
             <size>
              <width>5</width>
              <height>16</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="parameterSetting">
     <property name="title">
      <string>存储参数设置</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0" colspan="2">
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="topMargin">
         <number>15</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <property name="spacing">
           <number>20</number>
          </property>
          <property name="bottomMargin">
           <number>15</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
            <property name="spacing">
             <number>2</number>
            </property>
            <property name="sizeConstraint">
             <enum>QLayout::SetFixedSize</enum>
            </property>
            <item>
             <widget class="QLabel" name="filtTypeLabel">
              <property name="text">
               <string>文件类型</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="fileTypeBox">
              <item>
               <property name="text">
                <string>NonuniformityCorrection</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>BlindPixelRemoval</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
            <property name="spacing">
             <number>2</number>
            </property>
            <item>
             <widget class="QLabel" name="bandTypeLabel">
              <property name="text">
               <string>波段类型</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="bandTypeBox">
              <item>
               <property name="text">
                <string>Blue</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Green</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Red</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <property name="spacing">
           <number>20</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1">
            <property name="spacing">
             <number>25</number>
            </property>
            <item>
             <widget class="QLabel" name="gainLabel">
              <property name="text">
               <string>增益</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="gainBox">
              <item>
               <property name="text">
                <string>1x</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>2x</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>10x</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>30x</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,1">
            <property name="spacing">
             <number>2</number>
            </property>
            <item>
             <widget class="QLabel" name="timeLabel">
              <property name="text">
               <string>积分时间</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="timeBox">
              <item>
               <property name="text">
                <string>1ms</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>5ms</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>10ms</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>15ms</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>20ms</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>30ms</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>60ms</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="1,0,0,1">
     <property name="spacing">
      <number>20</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okBnt">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelBnt">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelBnt</sender>
   <signal>clicked()</signal>
   <receiver>saveFile</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>307</x>
     <y>266</y>
    </hint>
    <hint type="destinationlabel">
     <x>259</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
